borsh/src/schema.rs

```diff
@@ -651,7 +651,11 @@ macro_rules! impl_tuple {
 
         fn declaration() -> Declaration {
             let params = vec![$($name::declaration()),+];
-            format!(r#"Tuple<{}>"#, params.join(", "))
+            if params.len() == 1 {
+                format!(r#"({},)"#, params[0])
+            } else {
+                format!(r#"({})"#, params.join(", "))
+            }
```

Comment:
Do we really need it? I mean, in Rust and Python tuples require to have this comma to disambiguate from just grouping expressions with parenthesis `(2+2)*2`, but I don't see a reason to have it here. Do I miss something?

Reply:
This comma looks really jarring, and it may cause one to reconsider using the type of first element directly instead of one-element tuple).
Having it exactly the same as in rust has a benefit of possibility to parse it with `syn`, though parsing shouldn't be needed at all, as the same info is available via `schema::Definition`. 

----
the change was primarily to help visually distinguish declarations of tuples from what would be derived for a generic struct named `Tuple<T0, T1,...>`